### PR TITLE
calendar was added

### PR DESCRIPTION
**Pull request by ghost**
[DEMO LINK](https://Maksym845.github.io/layout_calendar/)
[TEST REPORT LINK](https://Maksym845.github.io/layout_calendar/report/html_report/)

### CODE REVIEW

File: src/styles/blocks/calendar.scss [hunk 1]
```diff
@@ -0,0 +1,68 @@
+@import "../utils/mixins";
+@import "../utils/variables";
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  width: 706px;
```
Comment (Dikey945) [inline src/styles/blocks/calendar.scss:8]: Place these values in variables.

File: src/styles/blocks/calendar.scss [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+  &__day {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+
+    @include square;
+
+    font-family: Arial, Helvetica, sans-serif;
```
Comment (Dikey945) [inline src/styles/blocks/calendar.scss]: This property is better write in HTML tag.

File: src/styles/blocks/calendar.scss [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+    font-size: 30px;
+
+    background-color: #eee;
+    border: 1px solid black;
```
Comment (Dikey945) [inline src/styles/blocks/calendar.scss:18]: Colors better to place in variables.

File: src/styles/blocks/calendar.scss [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+      }
+    }
+  }
+
+  &--month-length--30 {
+    .calendar__day--31 {
+      display: none;
+    }
+
+  }
```
Comment (Dikey945) [inline src/styles/blocks/calendar.scss]: What is the reason for these loops? All these rules can be written in one @for loop.

File: src/styles/blocks/calendar.scss [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+    background-color: #eee;
+    border: 1px solid black;
+
+    cursor: pointer;
+    transition: background-color 0.5s;
+  }
+
+  &__day:hover {
+    transform: translateY(-20px);
+    background-color: pink;
```
Comment (Dikey945) [inline src/styles/blocks/calendar.scss:26]: Place color in variable.

Comment (ghost) [reply src/styles/blocks/calendar.scss:26]: I`m using this color one time in this block, I think that I must not place this color in the variable.

Comment (ghost) [reply src/styles/blocks/calendar.scss:26]: Thank you for review of my code!

File: src/styles/blocks/calendar.scss (hunk 4 above)
Comment (ghost) [reply src/styles/blocks/calendar.scss]: This is a correct tip, thank you, I forgot about this.

File: src/styles/blocks/calendar.scss (hunk 5 above)
Comment (Dikey945) [reply src/styles/blocks/calendar.scss:26]: > I`m using this color one time in this block, I think that I must not place this color in the variable.

In the future, you will have many pages where this color can be reused) I just tell you mentor remarks that I get))) 

You are welcome!)

File: src/index.html [hunk 6]
```diff
@@ -1,12 +1,44 @@
...
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="page__body">
+    <div class="calendar calendar--start-day--sun calendar--month-length--30">
+      <div class="calendar__day calendar__day--1"></div>
```
Comment (Anna-Tkachenko) [inline src/index.html]: use nth-child in css instead of creating separate class for each day

**Review by Anna-Tkachenko — CHANGES_REQUESTED** [review]

File: src/styles/utils/_variables.scss [hunk 7]
```diff
@@ -0,0 +1,7 @@
+$margins:
+  "tue" 101px,
+  "wed" 202px,
+  "thu" 303px,
+  "fri" 404px,
+  "sat" 505px,
+  "sun" 606px;
```
Comment (MaksimusCRS) [inline src/styles/utils/_variables.scss:7]: Better to add formula, to calculate px

**Review by MaksimusCRS — APPROVED** [review]
GJ

**Pull request closed without merging**